iputils.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: iputils
+  version: '20221126'
```

Comment:
Oh wait hmm... This might need to be 0_ prefixed.

Reply:
Yeah I think we need to start this with 0.

https://github.com/wolfi-dev/os/blob/main/minio.yaml#L5

Reply:
I just saw the argon.yaml which uses this syntax: _https://github.com/wolfi-dev/os/blob/main/argon2.yaml#L3_

So, should I fix also the argon package version?

Reply:
I added '0.' prefix to the version.

Reply:
this change caused an error like this: 
_https://github.com/wolfi-dev/os/actions/runs/5287579349/jobs/9568238586?pr=2853#step:5:10_